source/sessions/driver-sessions.rst

```diff
@@ -1069,6 +1069,8 @@ ensure that they close any explicit client sessions and any unexhausted cursors.
     * Iterate through enough documents (3) to force a ``getMore``
     * Assert that the server receives a non-zero lsid equal to the lsid that ``find`` sent.
 
+11. For drivers that support forking, test that the session pool can be cleared after a fork
+    without calling ``endSession``.
```

Comment:
E.g., create ClientSession, record its lsid, delete it (so the lsid is pushed into the pool). Fork. In the parent, create a ClientSession and assert its lsid is the same. In the child, create a ClientSession and assert its lsid is different.